File: app/src/main/java/com/base/basepedo/service/BootCompleteReceiver.java
package com.base.basepedo.service;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

/**
 * 开机完成广播
 *
 * Created by xf on 2016/3/1.
 */
public class BootCompleteReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        Intent i = new Intent(context, StepService.class);
        context.startService(i);
    }
}

File: app/src/main/java/com/base/basepedo/service/BootCompleteReceiver.kt
package com.base.basepedo.service

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

/**
 * 开机完成广播
 *
 * Created by base on 2016/3/1.
 */
class BootCompleteReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        val i = Intent(context, StepService::class.java)
        context.startService(i)
    }
}
